doc/howto/network_bridge_resolved.md

```diff
@@ -35,7 +35,7 @@ DNS domain
 
   If this option is not set, the default domain name is `lxd`.
 
-  When specifying the DNS domain, prefix the domain name with `~`.
+  When configuring `resolved` with the DNS domain name, prefix the name with `~` (for example, if the DNS domain name is `lxd`, use `~lxd` to replace `<dns_domain>` in the following commands).
```

Comment:
Possibly whats caused some confusion is that below the command format is:

```
resolvectl domain <network_bridge> <dns_domain>
```

And then the example is:

```
resolvectl domain lxdbr0 '~lxd'
```

Suggesting that `<dns_domain>` is `~lxd`.

Perhaps we should change the command definition to:

```
resolvectl domain <network_bridge> ~<dns_domain>
```

Reply:
Yeah I think it may cause more people who are skip reading through it to sit up and notice , and then maybe go and read the change you've added already in more detail.